tests/test_subtests.py

```diff
@@ -593,7 +593,7 @@ def test_foo(subtests):
                 assert False
 
             with subtests.test("sub2"):
-                pass
+                assert False
```

Comment:
This shouldn't get executed.  Currently it's not getting printed in the report b/c it was passing.